### PR TITLE
[RAPPS-DB] Add cheat engine (cheatengine) 6.2

### PR DESCRIPTION
**Pull request by amir16yp**
Adds cheat engine 6.2. was tested on reactos nightly and was working on way earlier versions of ROS

### CODE REVIEW

**Comment by amir16yp** [comment]
i messed up the the line endings again

**Comment by learn-more** [comment]
> i messed up the the line endings again

The last line does not have a return.

File: cheatengine.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Cheat Engine
+Version = 6.2
+License = Freeware
+Description = Cheat Engine is an open-source tool for modifying game variables by scanning and altering memory addresses.
```
Comment (binarymaster) [inline cheatengine.txt:5]: You say in description that it's open source, but license is incorrectly set to "Freeware".

Comment (amir16yp) [reply cheatengine.txt:5]: I looked it up right now and apparently it is "source available" but no specific license

Comment (amir16yp) [reply cheatengine.txt:5]: well they do accept pull requests so i guess we can call it open source?
https://github.com/cheat-engine/cheat-engine

Comment (binarymaster) [reply cheatengine.txt]: https://github.com/cheat-engine/cheat-engine/issues/60#issuecomment-467179943

File: cheatengine.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [Section]
 Name = Cheat Engine
 Version = 6.2
-License = Freeware
+License = Open source
```
Comment (binarymaster) [inline cheatengine.txt:4]:
```suggestion
LicenseType = 1
```
Per https://reactos.org/wiki/RAPPS#LicenseType

**Review by learn-more — APPROVED** [review]

**Review by julenuri — APPROVED** [review]

**Comment by learn-more** [comment]
Thanks for your contribution!

**Pull request merged**